yaml-tests/src/test/resources/uuid-proto.yamsql

```diff
@@ -0,0 +1,57 @@
+#
+# uuid-proto.yamsql
+#
+# This source file is part of the FoundationDB open source project
+#
+# Copyright 2021-2025 Apple Inc. and the FoundationDB project authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+setup:
+  connect: "jdbc:embed:/__SYS?schema=CATALOG"
+  steps:
+    - query: drop schema template if exists uuid_proto_template
+    - query: drop database if exists /FRL/UUID_PROTO_YAML
+    - query: create database /FRL/UUID_PROTO_YAML
+    - load schema template: UUID_PROTO_TEMPLATE from com.apple.foundationdb.relational.yamltests.generated.uuidtests.UuidProtoTestsProto
+    - query: create schema /FRL/UUID_PROTO_YAML/UUID_PROTO_SCHEMA with template UUID_PROTO_TEMPLATE
+    - set schema state: "{\"name\": \"UUID_PROTO_SCHEMA\", \"database_id\": \"/FRL/UUID_PROTO_YAML\", \"template_name\": \"UUID_PROTO_TEMPLATE\", \"store_info\" : {\"formatVersion\": 2}}"
+---
+setup:
+  connect: "jdbc:embed:/FRL/UUID_PROTO_YAML?schema=UUID_PROTO_SCHEMA"
+  steps:
+    - query: INSERT INTO T1(ID, COL1, COL2("most_significant_bits", "least_significant_bits")) VALUES
```

Comment:
It would probably be good to have an `INSERT` statement that didn't try to set the UUID field at all. Something like:

```sql
INSERT INTO T1(ID, COL1) VALUES
    (8, 9)
```

And then validated that selecting this record results in a null field for the uuid, and that that works correctly.

Reply:
done!